src/ui/server/controllers/insurance/your-buyer/index.ts

```diff
@@ -25,7 +25,7 @@ export const get = async (req: Request, res: Response) => {
 };
 
 export const post = async (req: Request, res: Response) => {
-  const countries = await api.external.getCountries();
+  const countries = await api.keystone.countries.getAll();
```

Comment:
Sorry didn't spot this in the last PR but we only need to get countries if there are validation errors

Because we are only rendering the countries here if there are validation errors

Reply:
yes make sense